app/views/matches/_confirmed.html.erb

```diff
@@ -16,3 +16,7 @@
     </p>
   </div>
 <% end %>
+
+<p class="mt-4">
+  En cas d'empêchement, vous pouvez joindre le centre de vaccination au numéro suivant : <a href="tel:<%= @match.vaccination_center.phone_number %>"><%= @match.vaccination_center.phone_number %></a>
```

Comment:
Better with a link! 🙌
Maybe a `link_to` is more the rails way to do links 🙂
This should help you https://stackoverflow.com/questions/38821059/what-is-the-rails-way-equivalent-of-mailto-and-tel

Reply:
Thank you for the tip ! :)